README.md

```diff
@@ -102,3 +102,52 @@ const InstanceLogger = init(({ name }) => {
 
 const george = InstanceLogger({ name: 'George' }); // Created new instance: George
 ```
+
+
+### overrides()
```

Comment:
I would avoid word `options` as not everyone is aware that first argument is called like that. Using `props` as a part of name feels weird too since it's mostly term introduced by `stampit`. I think it should surely contain word _assign_ so it's clear what it does, perhaps `assignInstance` or even `assignToInstance` or `instanceAssign`.